### ISSUE TITLE
bring back thumbnail of panels?

### ISSUE THREAD

**Issue by evabenitoembo** [substance/texture#1139]
I would prefer if a thumbnail carousel of some sort would provide preview of what the panels are. In first demo, it was more intuitive. Maybe similar but horizontal (top?)

**Comment by michael** [comment]
We tried this but realised there's a number of problems with the thumbnails on top approach:

- does not scale (we would need to introduce a complex carousel navigation if there is like more than 3 sub-figures)
- images could have different aspect ratio, which makes it hard to position properly

So that's why we went for a space-saving minimal solution, with the only problem that you don't have random access anymore. Is that a requirement?

To address random access we would propose to allow expand the figure, to see all panels at once during editing. We thought it might be good to have that as a default even especially for editing figure packages which don't have a lot text content anyway.

**Comment by source-data** [comment]
yes, the scalability of thumbnails is an issue. Maybe it helps to separate 2 issues: 1) provide a visual hint and overview that the figure as multiple panels; 2) provide random access. I think your solution for random access is very good. But before the figure is expanded there is probably the need to have a slightly more explicit hint that there are sveral panels and how many?

**Comment by oliver----** [comment]
Good suggestion!

**Comment by michael** [comment]
@source-data how about this approach?

![image](https://user-images.githubusercontent.com/2931/52580928-afd4b380-2e29-11e9-9903-b115113bcb30.png)

I'd then leave out the navigation controls completely and allow only collapsed and expanded modes.

Another option for SourceData, to generally have them always expanded?

**Comment by evabenitoembo** [comment]
I personally do not like the option above... I like the current implementation with the count and the arrows. Maybe this could be made a bit more explicit with some wording, like 'panel 1 of x'. And maybe the whole wording + arrows should be wrapped in something like a button or something that makes it more visible?
